packages/wrangler/src/deployment-bundle/call-capnp.ts

```diff
@@ -0,0 +1,26 @@
+import { spawnSync } from "node:child_process";
+import { isAbsolute, join } from "node:path";
+import { sync as commandExistsSync } from "command-exists";
+
+export function callCapnp(capnp_schemas: string[], capnp_src_prefix?: string) {
+	if (!commandExistsSync("capnp")) {
+		throw new Error(
+			"The capnp compiler is required to upload capnp schemas, but is not present."
+		);
+	}
+	const srcPrefix = capnp_src_prefix
+		? isAbsolute(capnp_src_prefix)
+			? capnp_src_prefix
+			: join(process.cwd(), capnp_src_prefix)
+		: process.cwd();
```

Comment:
I think this could be simpler, unless I am missing something:

```suggestion
	const srcPrefix = resolve(capnp_src_prefix ?? ".");
```

Reply:
Great catch - forgot that this handled absolute paths 🤦‍♀️